scipy/stats/_qmc.py

```diff
@@ -147,7 +171,11 @@ def scale(sample, l_bounds, u_bounds, reverse=False):
         return (sample - lower) / (upper - lower)
 
 
-def discrepancy(sample, iterative=False, method="CD", workers=1):
+def discrepancy(
+        sample: npt.ArrayLike,
+        iterative: bool = False,
+        method: str = "CD",
```

Comment:
Since there is a disagreement here, I am waiting for more input :-)

Reply:
Unless the type of this `method` parameter is very common in scipy I'd suggest to just directly assign it to the `method` parameter without the inclusion of any type aliases.

Reply:
To clarify what I ment. In a lot of stats functions we have a literal (`alternative`). I am suggesting we could use the same logic in such cases, hence have a common helper to import `literal` and other things. Otherwise we will add quite some boilerplate to a lot of files.

Reply:
Updated in f9eb736c78310d2f294d5b2b558d849a151a99e7. I didn't put the Literal in `_util` though, since the method arguments are quite specific to the discrepancy function